wp-content/plugins/jetpack/_inc/blocks/tiled-gallery/view.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('wp-dom-ready', 'wp-polyfill'), 'version' => '6582996bec80f8a10b22aa210ce110dd');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).